packages/react-composites/tests/browser/chat/app/fake-back-end/types.ts

```diff
@@ -42,3 +42,5 @@ export interface NetworkEventModel {
    */
   maxDelayMilliseconds?: number;
 }
+
+export type BasicChatMessage = Pick<ChatMessage, 'sender' | 'senderDisplayName' | 'content' | 'metadata'>;
```

Comment:
Let's keep the `FakeChatClient` totally isolated from e2e testing concerns.

This `BasicChatMessage` is related to how the tests want to initialize the fake. So move this to the test helpers etc.

See also my comment in `FakeChatClient.ts`